expr/verdict.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package expr
+
+import (
+	"fmt"
+
+	"github.com/google/nftables/binaryutil"
+	"github.com/mdlayher/netlink"
+	"golang.org/x/sys/unix"
+)
+
+// This code assembles the verdict structure, as expected by the
+// nftables netlink API.
+// For further information, consult:
+//  - netfilter.h (Linux kernel)
+//  - net/netfilter/nf_tables_api.c (Linux kernel)
+//  - src/expr/data_reg.c (linbnftnl)
+
+type Verdict uint64
+
+// Verdicts, as per netfilter.h.
+const (
+	Drop   Verdict = 0x00
+	Accept Verdict = 0x01
```

Comment:
You’re missing Stolen (2). Could you add a Verdict prefix (so as to avoid clashes with other types) and change this to use iota while at it?

```go
const (
  VerdictDrop Verdict = iota
  VerdictAccept
  VerdictStolen
  // …
)
```

Reply:
Also, we should get these constants into golang.org/x/sys/unix, but one step at a time :)

Reply:
I’m not sure, but would it hurt to provide it? Maybe it will be used at some point. The C headers have it, so I think it should be available in Go.